CHANGELOG.md

```diff
@@ -2,6 +2,33 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [2.3.0-smart](https://github.com/awslabs/fhir-works-on-aws-deployment/compare/v2.2.0-smart...v2.3.0-smart) (2021-12-13)
```

Comment:
Isn't it better to have a separate PR for 2.3.0 ? It would be easier to review the PR.

Reply:
I have only made changes to cloudformation/alarms.yaml file in this PR. I am okay with having a separate PR to pull the upstream if everyone is okay with that.

Reply:
I agree. We probably want to handle the upgrade separately from this task.

Reply:
sounds good to me. I have reverted the upstream pull and have only included my cloudwatch alarm change in this PR.